.github/actions/nm-install-test-whl/action.yml

```diff
@@ -51,7 +51,7 @@ runs:
         echo "whl=${WHL_BASENAME}" >> "$GITHUB_OUTPUT"
         pip3 install ${WHL}[sparse]
         # report magic_wand version
-        MAGIC_WAND=$(pip3 show nm-magic-wand-nightly | grep "Version" | cut -d' ' -f2)
+        MAGIC_WAND=$(pip3 show nm-magic-wand-nightly || "" | grep "Version" | cut -d' ' -f2) || echo "nightly not installed"
```

Comment:
The introduction of the `|| ""` before `grep` changes the results:

```shell
set -euo pipefail

MAGIC_WAND=$(pip3 show nm-magic-wand-nightly || "" | grep "Version" | cut -d' ' -f2) || echo "nightly not installed"

if [ -z "$MAGIC_WAND" ]; then echo 'empty'; else echo "MAGIC_WAND=$MAGIC_WAND"; fi
```

In that script with this additional change, `MAGIC_WAND` no longer holds just 'version', it holds the entire output from `pip show`:

```
MAGIC_WAND=Name: nm-magic-wand-nightly
Version: 0.2.2.20240520
Summary: SparseLinear layers
Home-page: https://github.com/neuralmagic/magic_wand.git
Author: Neural Magic
Author-email: support@neuralmagic.com
License: Neural Magic Community License
Location: /venv/lib/python3.10/site-packages
Requires: cmake, numpy, torch
Required-by:
```